dynamic-simulation/dynamic-simulation-api/src/test/resources/DynamicSimulationResult.json

```diff
@@ -1,5 +1,37 @@
 {
   "version" : "1.0",
   "isOK" : true,
-  "logs" : ""
+  "logs" : "",
+  "curves" : [ {
+    "name" : "curve1",
+    "curve" : {
+      "metadata" : {
+        "name" : "curve1",
```

Comment:
This field is redundant with the one in metadata?
Why not just:
```json
"curves" : [
    { 
      "metadata": {
      "name": "curve2", 
        ....
       }
       ...
   }
]
```

Reply:
Done

Reply:
Thanks.
Could you has a look at Sonar bug?

Reply:
Fix it